runners/reference/java/pom.xml

```diff
@@ -83,5 +102,14 @@
       <artifactId>slf4j-jdk14</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
```

Comment:
This kind of dependency is a red flag and we have a JIRA to eliminate all such dependencies unless they are used only for executing. I went back and commented on the original PR to ask. Since this is for parity it LGTM but both should be explained in comment.